pkg/apis/imageregistry/v1/types.go

```diff
@@ -125,6 +125,10 @@ type ImageRegistrySpec struct {
 	// to push new images or delete existing ones.
 	// +optional
 	ReadOnly bool `json:"readOnly"`
+	// Redirect controls whether to route all data through the Registry,
+	// rather than redirecting to the backend.
+	// +optional
+	Redirect bool `json:"redirect"`
```

Comment:
The current behavior defaults this to true, right?  So we can't change that default behavior.  This field needs to default to enabling redirect when it is not set.  (ie rename it to DisableRedirect)

Reply:
Yes, you're right. I'll rename it to `DisableRedirect `.